Sources/NetworkProtection/Diagnostics/NetworkProtectionError.swift

```diff
@@ -64,6 +64,8 @@ public enum NetworkProtectionError: LocalizedError, CustomNSError {
 
     // Auth errors
     case noAuthTokenFound
+    case keychainError
```

Comment:
This was written to not include the error because we already emit keychain errors elsewhere, but tbh I'm not happy with that, we should send the error info anyway - I can just foresee being frustrated later when this error type appears and I can't see what caused it.

I'll update this to add the specific error.